lib/Target/SubstraitPB/Export.cpp

```diff
@@ -879,6 +891,9 @@ SubstraitExporter::exportOperation(LiteralOp op) {
     intervalDaytoSecond->set_seconds(intervalSecond);
     literal->set_allocated_interval_day_to_second(
         intervalDaytoSecond.release());
+  } // `UUIDType`.
+  else if (auto uuidType = dyn_cast<UUIDType>(literalType)) {
+    literal->set_uuid(mlir::cast<StringAttr>(value).getValue().str());
```

Comment:
I think that this is only valid if we interpret the attribute value as a 16-byte array as is, not as a string in hex format (see above).